patroni/ha.py

```diff
@@ -1201,3 +1266,25 @@ def wakeup(self):
         no "active" leader watch request in progress.
         This usually happens on the master or if the node is running async action"""
         self.dcs.event.set()
+
+    def unique_name_for_remote_master(self):
+        return 'remote_master:{}'.format(uuid.uuid1())
+
+    def get_target_to_follow(self, config):
```

Comment:
Too generic name. It seems to be used only in the standby_cluster scenario and only for figuring out the node that is the remote master. So why not name it get_remote_master?

Reply:
Besides, it is used not only for finding node to follow, but also for finding the one to clone from.